modules/ROOT/pages/constraints/index.adoc

```diff
@@ -65,14 +60,11 @@ Queries attempting to do any of the following will fail:
 * Create new relationships without all the properties or where the combination of property values is not unique.
 * Remove one of the mandatory properties.
 * Update the properties so that the combination of property values is no longer unique.
-////
 
 
 [NOTE]
 ====
-Node key constraints, node property existence constraints, and relationship property existence constraints are only available in Neo4j Enterprise Edition.
-// TODO: Switch the row above to the one below when adding back relationship key and uniqueness constraints
-//Node key constraints, relationship key constraints, node property existence constraints, and relationship property existence constraints are only available in Neo4j Enterprise Edition.
+Node key constraints, relationship key constraints, node property existence constraints, and relationship property existence constraints are only available in Neo4j Enterprise Edition.
```

Comment:
```suggestion
Node key constraints, relationship key constraints, node property existence constraints, and relationship property existence constraints are not available in Neo4j Community Edition.
```

They are available on all Aura environments, as far as I know.

Reply:
I don't know, that part of the sentence was there before and I didn't change it (only added the new relationship key to the list)